### ISSUE TITLE
Add per-slide content translation

### ISSUE THREAD

**Issue by binford2k** [binford2k/showoff#843]
@raphink suggests the ability to add language translations per slide. For some workflows it might be easier to have all translations in a single place. We should evaluate the feasibility and consider adding it.

One way would be to add div tags with language classes, something like

```html
<p class="lang en">Some English text</p>
<p class="lang fr">Bonjour mes amis!</p>
```

**Comment by raphink** [comment]
Another possibility would be to use the `~~~` syntax:

```markdown
Common text.

~~~LANG:en~~~
Some English text
~~~ENDLANG~~~

~~~LANG:fr~~~
Bonjour les amis!
~~~ENDLANG~~~
```

**Comment by raphink** [comment]
@binford2k any thoughts on the `~~~LANG~~~` proposal?

**Comment by raphink** [comment]
Closed by #859?
